src/collectionExporter.ts

```diff
@@ -173,9 +176,12 @@ class CollectionExporter implements IExporter {
 					toUpdate[getPrimary(existing)] = diff;
 				}
 			} else {
-				toInsert.push(lr);
+				if (!opts.skipCreate) toInsert.push(lr);
 			}
 
+			if (opts.forceUpdate && !toUpdate[getPrimary(lr)]) toUpdate[getPrimary(lr)] = lr;
```

Comment:
If the above fails due to there not being any diff, why would this make any difference? Won't it just be updating something that is already existing and already has the same values?